_posts/2015-03-02-java-and-hipster.md

```diff
@@ -0,0 +1,129 @@
+---
+layout: post
+title: Java and Hipster
+author: juuso
+excerpt: First experiences as a new employee of Solita... and some Java Hipsterism
+---
+
+I joined Solita as a software designer early this year and was soon asked to write a blog post about my experiences starting at Solita as a youngish software designer. I was lucky to start at the same day with two great tech enthusiast, one of them being a seasoned senior software designer Arto Santala, who by the way already wrote [a great wrap up of his first experiences at Solita](http://dev.solita.fi/2015/02/16/microservices.html).
+
+All the new employees of Solita get a thorough introduction to the company and as a part of the introduction, the coders get to implement a little exercise application to familiarize themselves with a typical modern web application stack. Programmers do the exercise themselves but asking for help and discussing architectural decisions with each other and with more senior employees is highly encouraged. Of the three of us who started at the same day, me and Arto were going to be very involved in Java development which meant that we would be using a Spring based stack. The third guy got a chance to delve into the world of Clojure. Lucky him?
```

Comment:
New employees at/of Solita get a thorough introduction to the company, and as a part of that coders get to